### DISCUSSION TITLE
Welcome new Dataloader Maintainers

### DISCUSSION THREAD

**Discussion by leebyron** [graphql/dataloader#292] [Announcements]
# Welcome new reviewers!

Hey all, I'm thrilled to welcome a new class of maintainers to the Dataloader project. You can find them all within the @graphql/dataloader-maintainers team.

With new maintainers, I thought it might be helpful to stake a claim around a philosophy and vision. Discussion welcome!

# Project philosophy and vision

Dataloader is perma-software. It has one very specific focus and serves to perform that focus well. It also serves as a reference implementation for the Dataloader concept to be ported to other languages. We should not expect new features or an evolution of the API - at least not without serious discussion and contemplation.

Instead, our vision for Dataloader is that it continues to serve its focus and reference well:

* There is regular project grooming that needs to take place, for example ensuring our Typescript types and build tool dependencies remain aligned with the standards du jour.
* We can always improve documentation and ensuring that the most common questions are well answered. 
* There are examples within this repo which need to remain up to date and relevant with the tools people are using today.

**Comment by saihaj** [comment]
Next action items here https://github.com/graphql/dataloader/issues/297
